platform-core-module/src/main/java/com/softicar/platform/core/module/email/buffer/BufferedEmailSender.java

```diff
@@ -54,8 +56,17 @@ private void send(AGBufferedEmail email) {
 		try (DbTransaction transaction = new DbTransaction()) {
 			email.reloadForUpdate();
 			if (email.getSentAt() == null) {
-				Transport.send(createMimeMessage(email));
-				Log.finfo("Sent e-mail #%s.", email.getId());
+				MimeMessage mimeMessage = createMimeMessage(email);
+				// send e-mail
```

Comment:
```suggestion
				// send email
```
...that's the spelling we commonly use for English strings.